src/main/java/httpserver/HttpResponse.java

```diff
@@ -1,5 +1,6 @@
 package httpserver;
 
 public class HttpResponse{
-    public Integer status = 0;
+    public String status = "";
+    public String body = "";
```

Comment:
Should these default to an empty string?

Reply:
I changed the default `status` to be 400 and the `body` to be "Bad Request".

Reply:
👍 One thing I have come to find through my own experiences is that defaults can lead to issues later. Should `400` and `Bad Request` be the default? Consider the scenario where someone sets the status to `200`, but not the body. Is `Bad Request` still a sensible default?